presto-cassandra/pom.xml

```diff
@@ -266,6 +266,7 @@
                         <configuration>
                             <excludes>
                                 <exclude>**/TestCassandraIntegrationSmokeTest.java</exclude>
+                                <exclude>**/TestCassandraIntergrationMixedCase.java</exclude>
```

Comment:
Why excluding it in default profile?

Reply:
I think `TestCassandraIntegrationSmokeTest` is excluded from default profile since it takes longer to run, but that's not the case with `TestCassandraIntergrationMixedCase` then why to exclude it from default profile?